pkg/tnf/identifier/identifiers.go

```diff
@@ -611,6 +612,12 @@ var Catalog = map[string]TestCatalogEntry{
 	},
 }
 
+// CommandIdentifier is  the Identifier used to represent the generic command test case.
+var CommandIdentifier = Identifier{
```

Comment:
This should be added into the catalog map. Give it a good description that explains what use case this handler is for, i.e. can be used with any command and would match any output. The caller is responsible for interpreting the output and extracting data from it.